src/Microsoft.ML.Transforms/Text/NgramHashingTransformer.cs

```diff
@@ -893,10 +908,10 @@ public sealed class ColumnInfo
             /// Describes how the transformer handles one column pair.
             /// </summary>
             /// <param name="name">Name of the column resulting from the transformation of <paramref name="inputColumnNames"/>.</param>
-            /// <param name="inputColumnNames">Name of the columns to transform. </param>
+            /// <param name="inputColumnNames">Names of the columns to transform. </param>
             /// <param name="ngramLength">Maximum ngram length.</param>
             /// <param name="skipLength">Maximum number of tokens to skip when constructing an ngram.</param>
-            /// <param name="allLengths">"Whether to store all ngram lengths up to ngramLength, or only ngramLength.</param>
+            /// <param name="allLengths">Whether to store all ngram lengths up to ngramLength, or only ngramLength.</param>
```

Comment:
>ngramLength [](start = 82, length = 11)

<cref>?

Reply:
I mean <cref> the reference

Reply:
Thank you!

---
In reply to: [258308077](https://github.com/dotnet/machinelearning/pull/2648#discussion_r258308077) [](ancestors = 258308077)